src/main/java/Main.java

```diff
@@ -0,0 +1,44 @@
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.Fruit;
+import service.FruitStrategy;
+import service.ParserService;
+import service.ReaderService;
+import service.ReportService;
+import service.WriterService;
+import service.impl.FruitStrategyImpl;
+import service.impl.ParserServiceImpl;
+import service.impl.ReaderServiceImpl;
+import service.impl.ReportServiceImpl;
+import service.impl.WriterServiceImpl;
+import strategy.BalanceOperation;
+import strategy.OperationsStrategy;
+import strategy.PurchaseOperation;
+import strategy.ReturnOperation;
+import strategy.SupplyOperation;
+
+public class Main {
+    private static final String SOURCE_PATH = "src/main/java/resources/database.csv";
+    private static final String REPORT_PATH = "src/main/java/resources/report.csv";
```

Comment:
please fix the `csv` files location as mentioned in common mistakes:
```suggestion
    private static final String SOURCE_PATH = "src/main/resources/database.csv";
    private static final String REPORT_PATH = "src/main/resources/report.csv";
```
The `csv` files is not a `java` files.